### PR TITLE
Updated Notebooks

### PR DESCRIPTION
**Pull request by Saurav-D**
# Peer Review Information

Updated Notebooks for SynthDet public dataset download and according to restructured import.

# Pull Request Check List

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: notebooks/SynthDet_Statistics.ipynb [hunk 1]
```diff
@@ -594,7 +605,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.7.7"
```
Comment (BlairLee) [inline notebooks/SynthDet_Statistics.ipynb:608]: Do you update the version?

Comment (Saurav-D) [reply notebooks/SynthDet_Statistics.ipynb:608]: I was using python 3.7 conda env, maybe 3.7 defaults to 3.7.7

**Comment by BlairLee** [comment]
Can you update the SynthDet dataset download part (`SIM2SIM Predictions` Session)?

**Comment by Saurav-D** [comment]
> Can you update the SynthDet dataset download part (`SIM2SIM Predictions` Session)?

What do you mean by update?

**Comment by BlairLee** [comment]
> > Can you update the SynthDet dataset download part (`SIM2SIM Predictions` Session)?
> 
> What do you mean by update?

I believe there is a SynthDet public dataset download in `SIM2SIM Predictions` Session, which also needs to be updated.

**Review by BlairLee — APPROVED** [review]

**Pull request merged**